pkg/local_object_storage/metabase/delete_test.go

```diff
@@ -39,9 +39,9 @@ func TestDB_Delete(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, l, 1)
 
-	// try to remove parent unsuccessfully
+	// try to remove parent, should be no-op, error-free
```

Comment:
Have you checked that `control drop-object` is not broken?

Reply:
depends on parent object removal expectations

Reply:
checked, removes all the child objects on parent object removal